.expeditor/publish_to_chocolatey.ps1

```diff
@@ -0,0 +1,46 @@
+$ErrorActionPreference="stop"
+
+Write-Host "--- Fetching latest release data from omnitruck API"
+# TODO WARNING DOOM: make sure  v=latest after testing complete
+# For now, we're going to update this for individual runs in pipeline tests, specifying unpublished versions.
+$Uri = "https://omnitruck.chef.io/stable/chef-workstation/metadata?p=windows&pv=2016&m=x86_64&v=0.16.33"
+$releaseRecord = Invoke-RestMethod -Uri $Uri -Headers @{accept="application/json"} -ErrorAction Stop
+
+Write-Host "--- Copying templates locally"
+$tempDir = Join-Path $env:temp ([System.IO.Path]::GetRandomFileName())
+New-Item $tempDir -ItemType Directory -Force | Out-Null
+Copy-Item -Path "components/packaging/chocolatey/*" -Destination $tempDir -Recurse
+
+Write-Host "--- Copying templates locally"
+$files = Get-ChildItem  -Path $tempDir -Filter *.*  -Recurse | Where-Object {!$_.PSIsContainer} | Select-Object FullName
+ForEach ($file in $files) {
+    (Get-Content $file.FullName).
+        Replace('$url$', $releaseRecord.url).
+        Replace('$version$', $releaseRecord.version).
+        Replace('$checksum$', $releaseRecord.sha256) |
+        Set-Content $file.FullName
+
+    Write-Host "--- DEBUG - Patched " $file.FullName " follows:"
+    Get-Content $file.FullName | Write-Host
+}
+
+Write-Host "--- Publishing package"
+
+$valid_build_creator="Chef Expeditor"
+$pack_cmd = "choco pack $tempDir/chef-workstation.nuspec --version " + $releaseRecord.version"
+$publish_cmd = "choco push chef-workstation.($releaseRecord.version).nupkg --timeout 600"
+
+try {
+    if($env:BUILDKITE_BUILD_CREATOR -eq $valid_build_creator) {
+        Invoke-Expression $pack_cmd
+        if ($LASTEXITCODE -ne 0) { throw "unable to choco pack" }
```

Comment:
Can you update this to also print out the stdout/stderr from the choco command so we know what went wrong? Or is that already echoed by default?

Reply:
The output is included in the logs by default.